SPECS/bind/bind.spec

```diff
@@ -1,21 +1,14 @@
 Summary:        Domain Name System software
 Name:           bind
-Version:        9.16.3
-Release:        3%{?dist}
+Version:        9.16.15
+Release:        1%{?dist}
```

Comment:
Same question as with the previous spec: I don't think this change is part of this PR.